lab-michael/lib/router.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
You don't need this file any longer. Express is providing a Router instance for you.